scripts/us_census/pep/pep_by_asrh/preprocess.py

```diff
@@ -0,0 +1,1053 @@
+                                drop=True)
+    df["SV"] = df["SV"].map(sv_dict)
+    skipped_df["SV"] = skipped_df["SV"].map(sv_dict)
+    derived_cols = _get_mapper_cols_dict("county_1900_1999")
+    df[pop_cols] = df[pop_cols].apply(pd.to_numeric, errors='coerce')
+    data = None
+    for dsv, sv in derived_cols.items():
+        data = df[df["SV"].isin(sv)].reset_index(drop=True)
+        data["SV"] = dsv
+        data = data.groupby(['Year', 'Location', "SV"]).sum().reset_index()
+        df = pd.concat([df, data])
+    df = pd.concat([df, skipped_df])
+    df = df.dropna()
+    df = _unpivot_df(df, ["Year", "Location", "SV"],
+                     pop_cols,
+                     default_col="Age")
+    # Creating SV's name using SV, Age Column
+    df["SV"] = df.apply(lambda row: _create_sv(row.SV, row.Age), axis=1)
+    # Creating GeoId's for State FIPS Code
+    df["Location"] = "geoId/" + df["Location"].astype("str").str.pad(
+        width=5, side="left", fillchar="0")
+    # Deriving Measurement Method for the SV's
+    df = _add_measurement_method(df, "SV", "Measurement_Method")
+    df = df[["Year", "Location", "SV", "Measurement_Method", "Count_Person"]]
+    return df
+
+
+def _process_county_2000_2009(file_path: str) -> pd.DataFrame:
+    """
+    Returns the Cleaned DataFrame consists
+    county data for the year 2000-2009.
+
+    Args:
+        file_path (str): Input File Path
+
+    Returns:
+        pd.DataFrame: Cleaned DataFrame
+    """
+    df = _load_df(file_path, "csv", header=0, encoding="latin-1")
+    # Skipping Below Year Values
+    # 1 = 4/1/2000 resident population estimates base
+    # 12 = 4/1/2010 resident 2010 Census population
+    # 13 = 7/1/2010 resident population estimate
+    df = df[(~df["YEAR"].isin([1, 12, 13])) &
+            (df["AGEGRP"] != 99)].reset_index(drop=True)
+    df["YEAR"] = 1998 + df["YEAR"]
+    cols = list(df.columns)
+    # Mapping Dataset Headers to its FullForm
+    cols_dict = _get_mapper_cols_dict("header_mappers")
+    for idx, val in enumerate(cols):
+        cols[idx] = cols_dict.get(val, val)
+    df.columns = cols
+    df["STATE"] = df["STATE"].astype("str").str.pad(width=2,
+                                                    side="left",
+                                                    fillchar="0")
+    df["COUNTY"] = df["COUNTY"].astype("str").str.pad(width=3,
+                                                      side="left",
+                                                      fillchar="0")
+    df["Location"] = "geoId/" + df["STATE"] + df["COUNTY"]
+    # Deriving New Columns
+    derived_cols = _get_mapper_cols_dict("county_2000_2009")
+    df = _derive_cols(df, derived_cols)
+    cols = cols + list(derived_cols.keys())
+    df["Age"] = df["AGEGRP"].apply(_get_age_grp_county_2000_2009)
+    f_cols = [val for val in cols if "Hispanic" in val]
+    df = _unpivot_df(df, ["Year", "Location", "Age"], f_cols)
```

Comment:
This just looks like wrapper and not much helpful.

Reply:
Jehangir has a similar comment on the helper function (https://github.com/datacommonsorg/data/pull/684#discussion_r916097194). I agree it wasn't helpful creating helper function on top of single function call, but considering adding example for melt method everywhere which is redundant. So I feel keeping helper function would be better with example and actual function call at one place